python/cuml/cuml/dask/common/input_utils.py

```diff
@@ -146,14 +146,13 @@ def calculate_parts_to_sizes(self, comms=None, ranks=None):
             for idx, wf in enumerate(self.worker_to_parts.items())
         ]
 
-        sizes = self.client.compute(parts, sync=True)
+        wfs, sizes_futures = zip(*parts)
+        all_sizes = self.client.compute(sizes_futures, sync=True)
 
-        for w, sizes_parts in sizes:
+        for w, sizes_parts in zip(wfs, all_sizes):
             sizes, total = sizes_parts
             self.parts_to_sizes[self.worker_info[w]["rank"]] = sizes
 
-            self.total_rows += total
```

Comment:
I haven't looked at this code before, but grep shows what I _think_ are some callers referencing this attribute. I think we still need it, but there's probably a more idiomatic way of computing it.

Reply:
Whoops, I think I just messed something up when staging the commit. I'll revert this part.